feeds/crates/crates.go

```diff
@@ -52,6 +52,16 @@ func fetchPackages() ([]*Package, error) {
 
 type Feed struct{}
 
+func New(feedOptions feeds.FeedOptions) (*Feed, error) {
+	if feedOptions.Packages != nil {
```

Comment:
I wonder if we can extract the guts of this into `UnsupportedOption()` or similar func in `feeds`?  Maybe passing in a const/globalvar of what the calling feed type supports. Would mean only one func needs to be updated as potential feed options expand?

Reply:
Potentially, it might make sense for feeds to specify some map of supported fields with a helper function to check for these. But until more options are added I think the current implementation is sufficient. I'll leave it to whoever expands the Options to decide the best way to tackle this for a multi-option approach.

Reply:
Yes it's definitely something that could be implemented at a later stage, and in a non breaking manner!